Newsletters: add 218 (2022-09-21)

harding opened the pull request:
- [x] Lede, releases/RCs, topic links @harding 
- [x] Client/services section @bitschmidty 
- [x] Core Lightning 5581 @Xekyo

Xekyo commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -0,0 +1,95 @@
+
+- [Core Lightning #5581][] Add plugin notification topic "block_processed". FIXME:Xekyo
+
+- [Eclair #2418][] and [#2408][eclair #2418] add support for receiving
```
I assume that both of the PRs were meant to be linked to? 

```suggestion
- [Eclair #2418][] and [#2408][eclair #2408] add support for receiving
```

Xekyo commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -42,7 +42,22 @@ This week's newsletter FIXME
+  updates to the [Fedimint][] federated Chaumian e-cash project, including
+  Lightning Network support.
+
+- **Bitpay wallet improves RBF support:**
+  Bitpay [improves][bitpay 12051] its [existing][bitpay 11935] support for
+  sending [RBF][topic rbf] transactions.
```
The commit is not particularly well-explained, but I have the impression that this is a bug fix. It seems to me that if the RBF function was called on a multi-recipient transaction it would only send to a single recipient in the replacement transaction, and now they support bumping multi-recipien transactions. I'm not sure if that's worth mentioning, but if it is, we might want to give a slightly more specific description?

bitschmidty replied:
Yeah, that is my take as well. I mostly used the recent fix commit as a way to point out the RBF support that was added previously, but apparently I missed it back then. I pushed a small change to add to this description.

@harding Im cool with you dropping this entry, your call!

harding replied:
I'd slightly prefer to keep it in its current edited form.  I think it's ok in this section to give a hat tip to a provider implementing a useful feature without going into details about a specific bug in their implementation.

LarryRuane commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+  for how a trusted set up procedure could be combined with the proposed
+  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] opcode to implement
+  behavior similar to that proposed by [drivechains][topic sidechains].
+  Drivechains are a type of sidechain where miners are normally in
+  charge of keeping the sidechain funds secure (compared to full nodes
+  which are in charge of securing funds on Bitcoin's mainchain).  Miners
```
```suggestion
  Drivechains are a type of sidechain where miners are normally responsible
  for keeping the sidechain funds secure (in contrast to full nodes,
  which secure funds on Bitcoin's mainchain).  Miners
```

LarryRuane commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+  Drivechains are a type of sidechain where miners are normally in
+  charge of keeping the sidechain funds secure (compared to full nodes
+  which are in charge of securing funds on Bitcoin's mainchain).  Miners
+  attempting to steal drivechain funds are required to announce their
```
I don't understand the concept very well, but maybe:
```suggestion
  attempting to steal drivechain funds unavoidably reveal their
```

LarryRuane commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+  soft fork (see BIPs [300][bip300] and [301][bip301]), but a previous
+  post to the mailing list (see [Newsletter #190][news190 dc]) described
+  how some other flexible proposed additions to Bitcoin's contracting
+  language could also allow implementation of drivechains.
```
```suggestion
  language could also allow the implementation of drivechains.
```

LarryRuane commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+- [Eclair #2418][] and [#2408][eclair #2418] add support for receiving
+  payments sent with [blinded routes][topic rv routing].  A spender
+  creating a blinded payment isn't provided with the identity of the
+  node receiving the payment, which may improve privacy (especially when
+  used with [unannounced channels][topic unannounced channels]).
```
```suggestion
  node receiving the payment, which may improve privacy, especially when
  used with [unannounced channels][topic unannounced channels].
```

Xekyo commented on _posts/en/newsletters/2022-09-21-newsletter.md:
```diff
@@ -53,7 +53,8 @@ wallets and services.*
 - **Bitpay wallet improves RBF support:**
   Bitpay [improves][bitpay 12051] its [existing][bitpay 11935] support for
-  sending [RBF][topic rbf] transactions.
+  sending [RBF][topic rbf] transactions by better handling bumping of multiple
+  output transactions.
```
Sounds much better.
Perhaps:
```suggestion
  sending [RBF][topic rbf] transactions by better handling bumping of 
  transactions with multiple recipients.
```

harding commented:
Made edits (thanks @glozow @Xekyo @LarryRuane @bitschmidty), added lede, removed releases/RC section (I don't see anything), added topic links, and reviewed @bitschmidty and @Xekyo contributions (thanks!).  Thanks everyone!

bitschmidty commented:
Added fedimint signet/faucet news to the client service section

bitschmidty reviewed (commented):
ACK

bitschmidty commented:
Looking good! Thanks @harding @Xekyo @LarryRuane and @glozow for contributing this week! 🚀
